### PR TITLE
Combine All Pages Through Bootstrap

### PR DESCRIPTION
**Pull request by ike-elue**
Adds
- **test.html**, an html file that uses bootstrap to combine the 3 separate sites into one nice-ish looking site
- screenshots of new site
- targets

Updates
- removes old master pages with previous iteration
- style.css 
- index.html, to add a link to the new site

**NOTE**: This does not change anything drastically for the previous website

### CODE REVIEW

File: ikenna-portfolio/portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -4,20 +4,78 @@
...
+  margin-bottom: 45px;
+}
+
+.navbar {
+  border: 0;
+  border-radius: 0;
+  font-size: 12px;
+  letter-spacing: 5px;
+  margin-bottom: 0;
+  padding-top: 15px;
```
Comment (sliang3) [inline ikenna-portfolio/portfolio/src/main/webapp/style.css]: This could just be padding: 15px 0;

File: ikenna-portfolio/portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -4,20 +4,78 @@
...
+  padding-bottom: 15px;
+}
+
+.navbar-nav  li a:hover {
+  color: #1abc9c !important; /*Will prolly delete*/
```
Comment (sliang3) [inline ikenna-portfolio/portfolio/src/main/webapp/style.css]: Did you need the !important here? If so you should leave a comment why.

File: ikenna-portfolio/portfolio/src/main/webapp/test.html [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+      </p>
+      <br>
+      <p class="desc">Here's a link to my LinkedIn <a href="https://www.linkedin.com/in/ikenna-elue-a28798166/">page</a></p>
+      <p class="desc">Here's a link to my GitHub <a href="https://github.com/ike-elue">page</a></p>
+    </div>
+
+    <!-- Second Container -->
+    <div class="container-fluid bg-2 text-center">
+      <h3 class="margin">STEP So Far</h3>
+      <h3><u>TODO</u></h3>
```
Comment (sliang3) [inline ikenna-portfolio/portfolio/src/main/webapp/test.html]: Remove?

File: ikenna-portfolio/portfolio/src/main/webapp/test.html [hunk 4]
```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
```
Comment (sliang3) [inline ikenna-portfolio/portfolio/src/main/webapp/test.html]: Maybe you could consider renaming this to something more descriptive than test.html. Not blocking for this pull request though since the demo is tomorrow.

File: ikenna-portfolio/portfolio/target/portfolio-1/style.css [hunk 5]
```diff
@@ -4,20 +4,78 @@
...
+  
 .column {
   flex: 1 1 auto;
   padding: 5px;
 }
 
-#quote-container {
-  margin-top: 20px;
+.container-fluid {
+  padding-bottom: 70px;
```
Comment (sliang3) [inline ikenna-portfolio/portfolio/target/portfolio-1/style.css]: This could be changed to padding: 70px 0;

and similar change below.

File: ikenna-portfolio/portfolio/target/portfolio-1/index.html [hunk 6]
```diff
@@ -22,6 +22,7 @@ <h1>Ikenna Elue's Portfolio</h1>
       <br>
       <p>Click <a href="calisthenics.html">here</a> to see some calisthenics moves I'm working towards!</p>
       <p>Click <a href="projects.html">here</a> to see/read some cool projects I'm working on!</p>
+      <p>Click <a href="test.html">here</a> to see test.</p>
```
Comment (sliang3) [inline ikenna-portfolio/portfolio/target/portfolio-1/index.html]: Consider rewording this to be more descriptive.

File: ikenna-portfolio/portfolio/src/main/webapp/test.html [hunk 7]
```diff
@@ -0,0 +1,267 @@
...
+
+      "Here's a link to the <a href=\"https://arxiv.org/pdf/1703.07511.pdf\">paper</a>.<br> \
+      Here's a link to the pytorch <a href=\"https://pytorch.org/tutorials/advanced/neural_style_tutorial.html\">tutorial</a>.", 
+      
+      "Here's a link to the <a href=\"https://www.gamasutra.com/blogs/MichaelKissner/20151027/257369/Writing_a_Game_Engine_from_Scratch__Part_1_Messaging.php\">site</a> explaining the message system further."
+    ];
+
+    const quotes = [
+      "The absence of evidence is not the evidence of absence.", 
+      "Is mayonaise an instrument?",
```
Comment (sliang3) [inline ikenna-portfolio/portfolio/src/main/webapp/test.html]: nit: mayonnaise

File: ikenna-portfolio/portfolio/src/main/webapp/test.html [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+        <div class="column">
+          <img id="pro-image2" class="image-format" src="" alt="">
+        </div>
+        <div class="column">
+          <img id="pro-image3" class="image-format" src="" alt="">
+        </div>
+      </div>
+      <br>
+      <div class="row">
+        <div class="col-sm-8">
```
Comment (sliang3) [inline ikenna-portfolio/portfolio/src/main/webapp/test.html]: Do you need to define this class and col-sm-4? It doesn't seem like they're being used in css.

**Review by sliang3 — APPROVED** [review]

File: ikenna-portfolio/portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/src/main/webapp/style.css]: done

File: ikenna-portfolio/portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/src/main/webapp/style.css]: done, left comment on why (forgot to put it in initially)

File: ikenna-portfolio/portfolio/src/main/webapp/test.html (hunk 3 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/src/main/webapp/test.html]: done-ish, improvised because I want to complete the portion and signal to myself in the site to do it soon

File: ikenna-portfolio/portfolio/src/main/webapp/test.html (hunk 4 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/src/main/webapp/test.html]: will do, will probably become index.html unless I find out that it's not a good solution

File: ikenna-portfolio/portfolio/src/main/webapp/test.html (hunk 8 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/src/main/webapp/test.html]: "col-sm" is defined by Bootstrap to have in ratio columns. A problem I found while trying to use this was that the columns were touching which I didn't want them to so I reused the "column definition" I had previously.

File: ikenna-portfolio/portfolio/src/main/webapp/test.html (hunk 7 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/src/main/webapp/test.html]: done

File: ikenna-portfolio/portfolio/target/portfolio-1/index.html (hunk 6 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/target/portfolio-1/index.html]: done

File: ikenna-portfolio/portfolio/target/portfolio-1/style.css (hunk 5 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/target/portfolio-1/style.css]: done

File: ikenna-portfolio/portfolio/src/main/webapp/test.html [hunk 9]
```diff
@@ -0,0 +1,266 @@
...
+  <!-- Theme Made By www.w3schools.com - No Copyright -->
+  <title>Bootstap Portfolio</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+  <script type="text/javascript">
```
Comment (lupengscu) [inline ikenna-portfolio/portfolio/src/main/webapp/test.html]: In general, it's better to put js code in a separate file unless there are just a couple of lines.

**Review by lupengscu — APPROVED** [review]

File: ikenna-portfolio/portfolio/src/main/webapp/test.html (hunk 9 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/src/main/webapp/test.html]: done

File: ikenna-portfolio/portfolio/src/main/webapp/test.html (hunk 4 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/src/main/webapp/test.html]: done
